src/components/forms/Form/Form.stories.tsx

```diff
@@ -174,3 +175,45 @@ export const addressForm = (): React.ReactElement => (
     </Fieldset>
   </Form>
 )
+
+export const signInForm = (): React.ReactElement => (
+  <Form onSubmit={mockSubmit} large>
+    <Fieldset legend="Sign In">
+      <span>
+        or <a href="javascript:void(0);">create an account</a>
+      </span>
+
+      <Label htmlFor="username">Username or email address</Label>
+      <TextInput id="username" name="username" type="text" />
+      <Label htmlFor="password-sign-in">Password</Label>
+      <TextInput
+        id="password-sign-in"
+        name="password-sign-in"
+        type="password"
+      />
+    </Fieldset>
+  </Form>
+)
+
+export const passwordResetForm = (): React.ReactElement => (
+  <Form onSubmit={mockSubmit} large>
+    <Fieldset legend="Reset password">
+      <span>Please enter your new password</span>
+
+      <Label htmlFor="newPassword">New Password</Label>
+      <TextInput id="newPassword" name="newPassword" type="password" />
+      <Label htmlFor="confirmPassword">Confirm Password</Label>
+      <TextInput id="confirmPassword" name="confirmPassword" type="password" />
+      <p>
+        <a href="javascript:void(0);" title="Forgot username">
+          Forgot username?
+        </a>
+      </p>
+      <p>
+        <a href="javascript:void(0);" title="Forgot password">
+          Forgot password?
+        </a>
+      </p>
```

Comment:
I think these are meant to be part of the Sign In form :) this one should have the "Hide my typing" toggle & submit button

Reply:
SMH. I think I got really confused. Sorry I'll clean up here.

Reply:
no worries!